steam_app/SearchBar.swift

```diff
@@ -0,0 +1,21 @@
+//
+//  SearchBar.swift
+//  steam_app
+//
+//  Created by Khumoyiddin on 17/01/23.
+//
+
+import UIKit
+
+extension UISearchBar {
+    func configureCustomSearchBar(searchBar: UISearchBar) {
```

Comment:
Since this is an extension, you don't need to pass the search bar object, you could directly use self